assets/app.js

```diff
@@ -0,0 +1,33 @@
+/* global hoodie */
+
+document.body.addEventListener('submit', function (event) {
```

Comment:
Can this just be on the document? Be good to have comments explaining things like event listeners as I'm not sure super new people would understand what's going on here.